packages/playground/scripts/build-env.sh

```diff
@@ -30,6 +30,7 @@ case $MODE in
     RELAY_DOMAIN='wss://relay.dev.grid.tf'
     BRIDGE_TFT_ADDRESS=GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG
     STELLAR_NETWORK=test
+    MINTING_URL="https://alpha.minting.tfchain.grid.tf"
```

Comment:
You can set the variable at the top with the default value if the URL will be the same on all of networks